libcxx/include/__ranges/range_adaptor.h

```diff
@@ -73,6 +81,13 @@ struct __range_adaptor_closure {
   }
 };
 
+#  if _LIBCPP_STD_VER >= 23
+namespace ranges {
+template <class _Tp>
+using range_adaptor_closure = __range_adaptor_closure<_Tp>;
```

Comment:
Since the standard doesn't specify this as an alias template, we probably need a better way to control its visibility.

Reply:
Can we make `range_adaptor_closure` a class template (as specified in the standard) and derived from `__range_adaptor_closure`? This is the approach of MSVC STL and seemingly working.

Reply:
Thanks! I've applied this approach in the latest commit.